cli_reference/basic_cli_operations.adoc

```diff
@@ -1,4 +1,4 @@
-= CLI Operations
+= Basic CLI Operations
```

Comment:
I don't think it makes sense to title the whole topic Basic CLI Operations while we have sections that are titled Common CLI Operations, Project Operations, etc.

It may be better to change the name in the nav bar to match the name of this topic, CLI Operations.